ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh

```diff
@@ -429,10 +429,17 @@ echo "${INSTANCE_ID}" >> "${SHARED_DIR}/aws-instance-ids.txt"
 
 BASTION_HOST_PUBLIC_DNS="$(aws --region "${REGION}" cloudformation describe-stacks --stack-name "${stack_name}" \
   --query 'Stacks[].Outputs[?OutputKey == `PublicDnsName`].OutputValue' --output text)"
+BASTION_HOST_PRIVATE_DNS="$(aws --region "${REGION}" cloudformation describe-stacks --stack-name "${stack_name}" \
+  --query 'Stacks[].Outputs[?OutputKey == `PrivateDnsName`].OutputValue' --output text)"
 
-echo "${BASTION_HOST_PUBLIC_DNS}" > "${SHARED_DIR}/bastion_dns"
-PROXY_URL="http://${PROXY_CREDENTIAL}@${BASTION_HOST_PUBLIC_DNS}:3128"
-echo "${PROXY_URL}" > "${SHARED_DIR}/public_proxy_url"
+echo "${BASTION_HOST_PUBLIC_DNS}" > "${SHARED_DIR}/bastion_public_address"
+echo "${BASTION_HOST_PRIVATE_DNS}" > "${SHARED_DIR}/bastion_private_address"
```

Comment:
Sorry, the saved data would be DNS or IP? If DNS, why both public DNS and private DNS?

Reply:
> the saved data would be DNS or IP

This depends on the ref author, any one is okay, in aws case, it is DNS.

> both public DNS and private DNS
Different cases use different items, inside cluster, the private one should be used, outside cluster, the public one should be used.